fullstack-cert/react-projects/tic-tac-toe/index.jsx

```diff
@@ -0,0 +1,84 @@
+export const Square = ({ value, onClick }) => {
```

Comment:
looking at the current react implementation on learn, I don't know if we are going to be able to export multiple child components into a parent component. 

right now, it looks like we just have one `index.jsx` file and the parent and child components are all in that file.
So we might need to remove these export keywords for all of the child components.